modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2651,21 +2652,34 @@ export class Wallet implements IWallet {
    * @param params signing options
    */
   private async signMessageTss(params: WalletSignMessageOptions = {}): Promise<SignedMessage> {
+    assert(params.reqId);
+
     if (!params.messagePrebuild) {
       throw new Error('messagePrebuild required to sign message with TSS');
     }
 
+    let txRequest;
     if (!params.messagePrebuild.txRequestId) {
-      throw new Error('txRequestId required to sign message with TSS');
+      const intentOption: IntentOptionsBase = {
+        reqId: params.reqId,
+        intentType: 'signmessage',
+        isTss: true,
+      };
+      txRequest = await this.tssUtils!.createTxRequestWithIntentForMessageSigning(intentOption);
+      params.messagePrebuild.txRequestId = txRequest.txRequestId;
     }
 
     if (!params.prv) {
       throw new Error('prv required to sign message with TSS');
     }
 
     try {
+      if (!txRequest) {
```

Comment:
this might be more readable if its sets up with 
if there is not txRequestId -> createTxRequestWithIntenForMessageSigning
else -> getTxRequest